packages/documentation/src/stories/components/accordion/accordion.stories.ts

```diff
@@ -16,6 +16,7 @@ const meta: MetaComponent<HTMLPostAccordionElement & HTMLPostCollapsibleElementE
   },
   args: {
     multiple: false,
+    headingLevel: 3,
```

Comment:
Is this the default value of the headingLevel/heading-level? Because there is no default value defined in the component. And also this value is not shown as selected in the controls table on the docs page...

Reply:
I would prefer not to have a default value here to prepare for the future when this property will be mandatory.

The value (3) is now selected in the control.

Reply:
I'm fine with that!